spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/AbstractMessageBusBinderPlugin.java

```diff
@@ -122,8 +130,14 @@ private void startTapListener(CuratorFramework client) {
 	 *
 	 * @param module the module whose consumer and producers to bind to the {@link MessageBus}.
 	 */
-	protected final void bindConsumerAndProducers(Module module) {
+	protected final void bindConsumerAndProducers(final Module module) {
+		String xdHistory = module.getDeploymentProperties() != null
+				? module.getDeploymentProperties().get(BusProperties.TRACK_HISTORY)
```

Comment:
Since this is an always supported property, should it have its dedicated smart getter in ModuleDeploymentProperties?